x/ccv/provider/keeper/keeper.go

```diff
@@ -779,7 +773,7 @@ func (k Keeper) DeleteSlashAcks(ctx sdk.Context, chainID string) {
 
 // AppendSlashAck appends the given slash ack to the given chain ID slash acks in store
 func (k Keeper) AppendSlashAck(ctx sdk.Context, chainID,
-	ack string, // TODO: consumer cons addr should be accepted here, see https://github.com/cosmos/interchain-security/issues/728
+	ack string,
```

Comment:
See https://github.com/cosmos/interchain-security/issues/728#issuecomment-1620706001 and comment that's being deleted
```suggestion
	ack types.ConsumerConsAddress, 
```